docs/Getting Started/Getting-Started-From-Scratch.md

```diff
@@ -1,108 +1,1150 @@
 # Getting started from scratch
+
+This section details easy to follow step by step instructions of how to setup one or more Quorum nodes from scratch for all new starters.
+
+Let's go through step by step instructions to setup a Quorum node with Raft consensus.
+
 ## Quorum with Raft consensus
-1. Build Quorum as described in the [getting set up](../Setup%20Overview%20%26%20Quickstart) section. Ensure that PATH contains geth and bootnode
+1. On each machine build Quorum as described in the [getting set up](../Setup%20Overview%20%26%20Quickstart) section. Ensure that PATH contains geth and bootnode
+    ```
+    $ git clone https://github.com/jpmorganchase/quorum.git
+    $ cd quorum
+    $ make all
+    $ export PATH=$(pwd)/build/bin:$PATH
+    ```
+
 2. Create a working directory which will be the base for the new node(s) and change into it
-3. Generate one or more accounts for this node using `geth --datadir new-node-1 account new` and take down the account address. A funded account may be required depending what you are trying to accomplish
+    ```
+    $ mkdir fromscratch
+    $ cd fromscratch
+    $ mkdir new-node-1
+    ```
+3. Generate one or more accounts for this node and take down the account address. A funded account may be required depending what you are trying to accomplish
+    ```
+    $ geth --datadir new-node-1 account new
+    
+     INFO [06-07|14:52:18.742] Maximum peer count                       ETH=25 LES=0 total=25
+     Your new account is locked with a password. Please give a password. Do not forget this password.
+     Passphrase: 
+     Repeat passphrase: 
+     Address: {679fed8f4f3ea421689136b25073c6da7973418f}
+    ```
+
+    !!! note 
+        You could generate one or more accounts (funded) accounts for additional node you want to start in advance and add them to genesis.json file as well
+    
+    
 4. Create a `genesis.json` file see example [here](../genesis). The `alloc` field should be pre-populated with the account you generated at previous step
-5. Generate node key `bootnode --genkey=nodekey` and copy it into datadir
-6. Execute `bootnode --nodekey=new-node-1/nodekey --writeaddress` and take note of the displayed output. This is the enode id of the new node
+    ```
+    $ vim genesis.json
+    ... paste below with account '679fed8f4f3ea421689136b25073c6da7973418f' created at previous step
```

Comment:
"... alloc account '679fed8f4f3ea421689136b25073c6da7973418f' created in the previous step and an additional account 'c5c7b431e1629fb992eb18a79559f667228cd055`"
...
```
"alloc": {
    "0x679fed8f4f3ea421689136b25073c6da7973418f": {
      "balance": "1000000000000000000000000000"
    },
    "0xc5c7b431e1629fb992eb18a79559f667228cd055": {
      "balance": "1000000000000000000000000000"
    }
  }
```